src/me.rs

```diff
@@ -284,6 +284,7 @@ impl MotionEstimationSubsets {
   }
 }
 
+#[allow(clippy::stable_sort_primitive)]
```

Comment:
> barrbrain> we don't need a stable sort for finding the median of primitive values

So these could actually be fixed instead of annotating them with allow

Reply:
Awesome. I've pushed a new commit which uses unstable sorting.